specification/loadtestservice/LoadTestService/tspconfig.yaml

```diff
@@ -19,14 +19,15 @@ options:
   #   azure-resource-provider-folder: "data-plane"
   #   examples-directory: ./examples
   "@azure-tools/typespec-python":
-    package-name: "azure-developer-loadtesting"
+    package-dir: "azure-developer-loadtesting"
+    package-name: "{package-dir}"
     # emitter-output-dir: "{python-sdk-folder}/sdk/{service-directory-name}/{package-name}"
     package-mode: dataplane
   "@azure-tools/typespec-ts":
     package-dir: "load-testing-rest"
     title: Azure Load Testing
     description: Azure Load Testing Client
-    generateMetadata: false
+    generateMetadata: true
     generateTest: false
     packageDetails:
       name: "@azure-rest/load-testing"
```

Comment:
please add emitter options for `@azure-tools/typespec-csharp`.
```
"@azure-tools/typespec-autorest":
     package-dir: "Azure.Developer.LoadTesting"
     clear-output-folder: true
     namespace: "{package-dir}"
```

Reply:
Updated.